python/ray/serve/schema.py

```diff
@@ -98,10 +108,11 @@ class Config:
             "('/tmp/ray/session_latest/logs/serve/...')."
         ),
     )
-    enable_access_log: bool = Field(
-        default=True,
+    access_log: Union[str, AcessLoggingType] = Field(
+        default="ALL",
         description=(
-            "Whether to enable access logs for each request. Default to True."
+            "Configure how to access the Serve logs. Default to 'ALL' to enable "
+            " streaming and file access logs together."
```

Comment:
Is this completely different from the `enable_access_log` field from before? I'm under the impression that "access logs" are the lines like
`(ServeReplica:default:Hello pid=15902) INFO 2023-11-07 13:10:11,907 Hello default#Hello#CvGNYA d7e93fde-3ec2-4172-8bbf-84377378a9d0 / default replica.py:730 - __CALL__ OK 92.1ms`

However it seems this new `access_log` field defines whether all Serve logs should be written to file and/or streamed to the driver.

Reply:
sync offline, reverted back to the original api. We will add more functionality as customer asked for it.